src/main.rs

```diff
@@ -63,13 +115,26 @@ fn main() {
             info!("Running pacdiff");
             detect();
         }
+        Operation::GenComp(gencomp_args) => {
+            info!("Generating shell completions for {}. Please pipe `stderr` to a file to get completions as a file, e.g. `ame gencomp fish 2> file.fish`", gencomp_args.shell);
```

Comment:
Why does this output to stderr? Wouldn't it be easier for the user to just specify a file to save to? something like 'ame gencomp file.fish'

Reply:
This way if the user has an existing competions file, it's much easier to just `2>> file`, and it lets the user how many / what the completions could also look like. It also lets the user pipe the output around into formatters, or even a shell `source` function (e.g., `ame gencomp bash 2> >(bash)` for testing

Reply:
But why does it output to stderr?

Reply:
Outputting to stdout would make it take our info prompt with it

Reply:
but why not make the info prompt output to stderr and the rest to stdout? i believe most people would do `ame gencomp fish > fish.file` first

Reply:
Because our info!() macro prints to stdout in a non-configurable way, and most users who are going to be generating their own shell completions aren't going to mind writing `2> file`

Reply:
Yeah that makes sense